src/datetime.ts

```diff
@@ -260,9 +261,19 @@ export function dateTimeExpr(d: DateTime | DateTimeExpr, normalize = false) {
     }
   }
 
+  const unitsString = units.join(', ');
+
+  if (toJSON) {
+    if (d.utc) {
+      return new Function(`return new Date(Date.UTC(${unitsString}))`)().toJSON();
```

Comment:
This looks odd to me. Why are we creating a function that we execute immediately? Isn't this essentially an eval?

@arvind